Lesson3/src/Homework.java

```diff
@@ -0,0 +1,194 @@
+     * - Print in console matrix with given size, but:
+     * - If remainder of the division array element by 3 is zero - print "+" sign instead of array element value
+     * - If remainder of the division array element by 7 is zero - print "-" sign instead of array element value
+     * - Print "*" otherwise
+     * <p>
+     * Example:
+     * - Values from console -  2 and 3
+     * - Array will look like this (values be different because it's a random)
+     * 6 11 123
+     * 1 14 21
+     * - For this values output in console should be:
+     * <p>
+     * + * *
+     * * - +
+     * <p>
+     * Note that 21 % 3 == 0 and 21 % 7 = 0, but output is not +-, but +
+     */
+
+    /** Version from Gleb     */
+//    public static void printMatrix() {
+//        Scanner scanner = new Scanner(System.in);
+//        Random random = new Random();
+//
+//        int height = scanner.nextInt();
+//        int width = scanner.nextInt();
+//        int[][] array = new int[height][width];
+//        for (int i = 0; i < array.length; i++) {
+//            for (int j = 0; j < array[i].length; j++) {
+//                array[i][j] = random.nextInt(100);
+//                System.out.print(array[i][j] + " ");
+//            }
+//            System.out.println();
+//        }
+
+
+//        for (int[] row : array) {
+//            for (int element : row) {
+//                if (element % 3 == 0) {
+//                    System.out.print("+");
+//                } else if (element % 7 == 0) {
+//                    System.out.print("-");
+//                } else {
+//                    System.out.print("*");
+//                }
+//            }
+//            System.out.println();
+//        }
+//    }
+
+    public static void printMatrix() {
+
+        /** - Read two ints from console                   */
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Input height: ");
+        int height = scanner.nextInt();
+
+        Scanner scanner1 = new Scanner(System.in);
+        System.out.print("Input width: ");
+        int width = scanner1.nextInt();
+
+        /** - Create a two-dimensional int array (use ints that you read from console height and width)         */
+
+        int[][] twoDArray = new int[height][width];
+
+        /** - Fill array with random values (under 100)         */
+
+        Random random = new Random();
+
+        for (int i = 0; i < twoDArray.length; i++) {
+            for (int j = 0; j < twoDArray[i].length; j++) {
+                twoDArray[i][j] = random.nextInt(100);
+                System.out.print(twoDArray[i][j] + " ");
+            }
+            System.out.println();   /** - Print in console matrix with given size:         */
+        }
+        /** - If remainder of the division array element by 3 is zero - print "+" sign instead of array element value */
+
+        for (int[] row : twoDArray) {
+            for (int element : row) {
+                if (element % 3 == 0) {
```

Comment:
this check is correct but you also need to do a check that element can be divided by 7, and in this case print "-"